.travis.yml

```diff
@@ -26,6 +27,12 @@ matrix:
       env: COMPOSER_FLAGS="--prefer-lowest"
     - php: 5.6
       env: DEPS=dev COMPOSER_FLAGS="--prefer-stable" SYMFONY_VERSION=3.0.*
+    - php: 5.6
+      env: DEPS=dev SYMFONY_VERSION=3.2.*
+    - php: 7.0
+      env: DEPS=dev SYMFONY_VERSION=3.2.*
+    - php: 7.1
+      env: DEPS=dev SYMFONY_VERSION=3.2.*
```

Comment:
These should be removed. We should only test one Symfony version against all PHP version and one PHP version against all Symfony versions.